src/me/StevenLawson/TotalFreedomMod/Commands/Command_smite.java

```diff
@@ -1,22 +1,24 @@
 package me.StevenLawson.TotalFreedomMod.Commands;
 
 import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.bukkit.ChatColor;
 import org.bukkit.GameMode;
 import org.bukkit.Location;
-import org.bukkit.World;
```

Comment:
@OnlineRepz Actually, the way that it has been done (If it works, I have not tested it) would be a cleaner way to do it.